backend/src/Edna/Upload/Web/API.hs

```diff
@@ -14,10 +15,12 @@ import Universum
 
 import Data.Aeson.TH (deriveToJSON)
 import Data.Swagger (ToSchema(..))
-import Fmt (Buildable(..))
-import Servant.API (Capture, JSON, Post, QueryParam, Summary, (:>))
+import Data.Text (unpack)
```

Comment:
I prefer `toString` because it's easier to understand what is converted to what (`unpack` also exists for bytestrings for example), but that's up to you.